examples/configurations/drivers/chargepoint1.config

```diff
@@ -0,0 +1,15 @@
+{
+    "driver_config": {
+        "stationID" : "1:34003",
+        "username" : "9d44ba0be5fe6b6628e50af1335d4fcf5743a6f3c63ee1464051443",
+        "password" : "**Put valid Chargepoint password here.**"
+    },
+    "campus": "campus",
+    "building": "building",
+    "unit": "station_001_01",
```

Comment:
The "campus", "building", and "unit" values are now ignored by the driver framework. We now use the name of the configuration file when it's put in the config store to derive the device topic .